src/test/java/uk/gov/hmcts/cmc/claimstore/models/sampledata/SampleCountyCourtJudgment.java

```diff
@@ -6,12 +6,13 @@
 import uk.gov.hmcts.cmc.claimstore.models.legalrep.StatementOfTruth;
 import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
 
+import java.math.BigDecimal;
 import java.time.LocalDate;
 
 public class SampleCountyCourtJudgment {
 
     private TheirDetails defendant = SampleTheirDetails.builder().individualDetails();
-    private double paidAmount = 0.0;
+    private BigDecimal paidAmount = BigDecimal.valueOf(0);
```

Comment:
There is a `BigDecimal.ZERO` constant, would be nice to make use of it here.

Reply:
ok